go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/wptide/wptide
+
+go 1.11
+
+require (
```

Comment:
This seems very explicit and not very tenable in my opinion. Seems like we have to track more dependencies than before.

Reply:
The extra dependencies there are a result of indirect and test dependencies being recorded. Its automatically updated (by default) when running commands such as `go build`, `go test` or `go mod tidy`, unless the flag `-mod=readonly` is used.

The idea is `go mod tidy` ensures your current `go.mod` reflects the dependency requirements for all possible combinations of OS, architecture, and build tags ([see docs for more info](https://github.com/golang/go/wiki/Modules#why-does-go-mod-tidy-record-indirect-and-test-dependencies-in-my-gomod)).